package.json

```diff
@@ -4,14 +4,14 @@
   "scripts": {
     "ng": "ng",
     "start": "node server.js",
-    "serve": "ng serve --configuration=dev",
-    "start:staging": "gulp set --env=staging && concurrently --kill-others \"node server.js\" \"npm run serve\"",
-    "start:prod": "gulp set --env=prod && concurrently --kill-others \"node server.js\" \"npm run serve\"",
+    "start:staging": "gulp set --env=staging && concurrently --kill-others \"node server.js\" \"ng serve --configuration=dev-staging\"",
+    "start:prod": "gulp set --env=prod && concurrently --kill-others \"node server.js\" \"ng serve --configuration=dev-production\"",
```

Comment:
@Nishanthh It should have the serve scripts bro
`"start:staging": "ng serve --configuration=dev-staging"

Reply:
@Nishanthh Also have you given the port value in the angular.json file? I don't see it there

Reply:
> @Nishanthh It should have the serve scripts bro
> `"start:staging": "ng serve --configuration=dev-staging"

@manzur i have added the scripts.

Reply:
`        "serve": {
          "builder": "@angular-devkit/build-angular:dev-server",
          "options": {
            "browserTarget": "civis:build",
            "port": 3400
          },
          "configurations": {
            "production": {
              "browserTarget": "civis:build:production"
            },
            "staging": {
              "browserTarget": "civis:build:staging"
            },
            "dev-staging": {
              "browserTarget": "civis:build:dev-staging"
            },
            "dev-production": {
              "browserTarget": "civis:build:dev-production"
            }
          }
        },`
@manzur This is how port added in angular.json file

Reply:
@Nishanthh I see `"start:staging": "gulp set --env=staging && concurrently --kill-others \"node server.js\" \"ng serve --configuration=dev-staging\"",` whereas it should be 
`"start:staging": "ng serve --configuration=dev-staging"`